_includes/footer.html

```diff
@@ -39,9 +39,9 @@
 <script src="{{ site.baseurl }}/resources/js/vendor/unslider.js" type="text/javascript"></script>
 
 <!-- Highlight -->
-<script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.9.0/highlight.min.js" type="text/javascript"></script>
-<script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.9.0/languages/scala.min.js" type="text/javascript"></script>
-<script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.9.0/languages/java.min.js" type="text/javascript"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.7.2/highlight.min.js" type="text/javascript"></script>
```

Comment:
It is difficult for me to verify that this doesn't break anything anywhere else.

Reply:
I need this upgrade to get support of `text` area.

![image](https://user-images.githubusercontent.com/13123162/117014870-84339180-acf1-11eb-8d0c-778d4a1aeede.png)
instead of
![image](https://user-images.githubusercontent.com/13123162/117014431-14250b80-acf1-11eb-9484-ddb514d0ea34.png)

I hope it does not break anything, but if it does we can revert it and find another solution.